### ISSUE TITLE
broot --install followed by br asks to install again

### ISSUE THREAD

**Issue by texastoland** [Canop/broot#837]
My `.zshrc` (on Mac) starts with:

```sh
[[ ! -e $XDG_CONFIG_HOME/broot/launcher/bash/br ]] && broot --install
source "$XDG_CONFIG_HOME/broot/launcher/bash/br"
```

But the first time I run `br` it asks:

```
Broot should be launched using a shell function.
This function most notably makes it possible to cd from inside broot
(see https://dystroy.org/broot/install-br/ for explanations).

Can I install it now? [Y/n]
```

The cause appears to be the implementation checking for `$XDG_CONFIG_HOME/broot/launcher/installed-vX` but `broot --install` doesn't include it.

**Comment by Canop** [comment]
This should be fixed by #842 

Tests welcome

**Comment by texastoland** [comment]
Trying now! I'll confirm within the PR 🙏🏼
